ms-Iot/contentP1

nbaulesglobalsolutions opened the pull request:
1st merge for basefork; ms-iot/content vs base:11_2_16_rollback and;

headfork;ms-iot/content vs. compare:develop

compared across forks

i have no idea what just happened. apologies.


nb

Commit:
i2c and spi code bugs

'I2cDevice' does not contain a definition for 'GetDefault', same is true for Spi

alecont committed:
Add initial doc pages for WiFi Direct

Commit:
Fixed syntax bug on ETW page which prevents users from seeing the full xperf and tracerpt commands

bfjelds committed:
Update SetupMBM to include BIOS settings

Commit:
added step in install

Clarified a reboot is required after installing ADK

stephenhuang committed:
Merge pull request #636 from chsha/patch-7

added step in install

alecont committed:
Add documentation for WiFi Direct

rachitb777 committed:
Fixing the community link in the footer

rachitb777 committed:
Merge pull request #637 from rachitb777/develop

Fixing the community link in the footer

rachitb777 committed:
Revert "Fixing the community link in the footer"

This reverts commit 22ef8bcd841dd6c54b67c8a2e43c6d78a9aa08ec.

rachitb777 committed:
Merge pull request #638 from rachitb777/develop

Revert "Fixing the community link in the footer"

rachitb777 committed:
Merge remote-tracking branch 'upstream-private/develop' into develop

Commit:
Removed tags in response to suggested fix

Commit:
Adjusted wording for computer vision page to more accurately reflect Cognitive Services

bfjelds committed:
remove extra </div>

rachitb777 committed:
Fix the DISM page

rachitb777 committed:
Merge pull request #639 from rachitb777/develop

Fix the DISM page

rachitb777 committed:
Add Dragonboard flashing tool documentation and note about mouse/keyboard not detected when micro USB connected

rachitb777 committed:
Merge pull request #640 from rachitb777/develop

Add Dragonboard flashing tool documentation and note about mouse/keyb…

rachitb777 committed:
Merge pull request #633 from derekameer/develop

Fixed syntax bug on ETW page which prevents users from seeing the ful…

rachitb777 committed:
Merge pull request #998 from bfjelds/develop

Update SetupMBM to include BIOS settings

mattwojo committed:
Update Community page link URL

Commit:
new footer link for community

rachitb777 committed:
Fixing broken links in DB410 getting started flow

rachitb777 committed:
Merge pull request #1004 from rachitb777/develop

Fixing broken links in DB410 getting started flow

deisterhold committed:
Fixed misspelling.

rachitb777 committed:
Merge pull request #1005 from deisterhold/patch-1

Fixed misspelling.

rachitb777 committed:
Merge pull request #1002 from alecont/develop

Add documentation for WiFi Direct

rachitb777 committed:
Merge remote-tracking branch 'upstream-private/develop' into develop

rachitb777 committed:
Merge branch 'patch-1' of https://github.com/cyberh0me/content into cyberh0me-patch-1

rachitb777 committed:
Addressing code-review feedback from bfjelds

rachitb777 committed:
Merge branch 'cyberh0me-patch-1' into develop

rachitb777 committed:
Merge pull request #1006 from rachitb777/develop

i2c and spi code bug

rachitb777 committed:
Fixing overlapping columns in FAQ page

rachitb777 committed:
Merge pull request #1008 from rachitb777/develop

Fixing overlapping columns in FAQ page

dlemson committed:
fixed spelling error

rachitb777 committed:
Some fixes in FAQ page

rachitb777 committed:
Merge pull request #1010 from rachitb777/develop

Some fixes in FAQ page

mattwojo committed:
Community link fix

jessekaplan committed:
Updating docs

alecont committed:
Merge pull request #1009 from dlemson/patch-1

fixed spelling error

Sneezry committed:
Update WebcamApp.md

Commit:
attempt to fix footer problem for homepage

rachitb777 committed:
Merge pull request #1013 from Sneezry/patch-1

Fix broken link

ski35off committed:
microUSB -> micro-USB

Micro-USB is the correct term according to www.usb.org.

bfjelds committed:
add sample doc for companion app

Commit:
corrected gigabit ethernet for Pi

bfjelds committed:
Merge pull request #1011 from jessekaplan/develop

Updating docs

bfjelds committed:
fix typo

bfjelds committed:
fix typo

bfjelds committed:
fix typo

Commit:
added apollo lake

bfjelds committed:
address some feedback

rachitb777 committed:
Merge pull request #1017 from chsha/patch-1

corrected gigabit ethernet for Pi

rachitb777 committed:
Merge pull request #1015 from ski35off/develop

microUSB -> micro-USB on SetupDB410c.htm

rachitb777 committed:
Merge pull request #1016 from bfjelds/develop

add sample doc for companion app

rachitb777 committed:
Merge pull request #1019 from chsha/patch-2

added apollo lake

rachitb777 committed:
Bug 9354440 Adding redirect on Commercialization page

rachitb777 committed:
Merge pull request #1021 from rachitb777/develop

Bug 9354440 Adding redirect on Commercialization page

rachitb777 committed:
Removing MSConnect page

rachitb777 committed:
Merge pull request #1022 from rachitb777/develop

Removing MSConnect page

Commit:
removed BIOS version. added note for common error

rachitb777 committed:
Adding redirect for SigninMSConnect page

rachitb777 committed:
Merge pull request #1024 from rachitb777/develop

Adding redirect for SigninMSConnect page

rachitb777 committed:
Merge pull request #1023 from chsha/patch-3

removed BIOS version. added note for common error

rachitb777 committed:
Fixing broken link in SerialUART sample

rachitb777 committed:
Merge pull request #1027 from rachitb777/develop

Fixing broken link in SerialUART sample

Commit:
Fixed unordered sub list for supported languages

IoTGirl committed:
Merge pull request #1028 from radu-matei/patch-1

Fixed unordered sub list for supported languages

kevinyr committed:
Update WhatsNew.md

The Get Started link text at the end of the Support for Intel Joule paragraph should not be capitalized.

rachitb777 committed:
Merge pull request #1029 from kevinyr/patch-2

Update WhatsNew.md

Commit:
Updating release notes to 14986

Updating release notes to 14986 - New SDS build.

rachitb777 committed:
Merge pull request #1030 from zeeshanfurqan/develop

Updating release notes to 14986

Commit:
Fix typo

increddibelly committed:
Update SerialUART.md

these line breaks were making the content hard to read.

pkuehne committed:
Update ConnectAppToCloud.md

Fixed typo

bfjelds committed:
fix "srvices" typo

rachitb777 committed:
Merge pull request #1032 from rogersachan/patch-1

Fixed typo in "Get Started"

rachitb777 committed:
Merge pull request #1033 from increddibelly/patch-2

Update SerialUART.md

rachitb777 committed:
Merge pull request #1035 from pkuehne/patch-1

Update ConnectAppToCloud.md

mattwojo committed:
Redirect IoTDashboardTroubleshooting.htm page

This IoTDashboardTroubleshooting page is no longer used though is discoverable so will now redirect

rachitb777 committed:
Merge pull request #1037 from mattwojo/develop

Redirect IoTDashboardTroubleshooting.htm page

MattGosline committed:
Add english version of VirtualAudioMicrophoneArray.md

MattGosline committed:
Fix typo in VirtualMicrophoneArrayDriver.md

IoTGirl committed:
Merge pull request #1038 from MattGosline/develop

Add VirtualAudioMicrophoneArray.md - Merged based on acceptance of https://github.com/ms-iot/samples/pull/393

Commit:
Fixing the Azure IoT Hub Device Management status

Fixing the Azure IoT Hub Device Management status - Removing "currently
in preview" text

rachitb777 committed:
Merge pull request #1040 from parameshbabu/develop

Fixing the Azure IoT Hub Device Management status

saylenty committed:
Fix typo in DeviceOptions.md

IoTGirl committed:
Merge pull request #1041 from saylenty/patch-1

Thanks for fixing the typo in DeviceOptions.md

Commit:
fixed typos to close issue #1039

Commit:
fixed code

mattwojo committed:
Set up redirects for all of the lang-loc pages to point back to en-us content

rachitb777 committed:
Merge pull request #1045 from lukas2005/patch-1

fixed code

rachitb777 committed:
Merge pull request #1043 from derekameer/develop

fixed typos to close issue #1039

rachitb777 committed:
Merge pull request #1046 from mattwojo/develop

Set up redirects for all lang-loc pages to point back to en-us content

mattwojo committed:
Remove all lang-loc files

rachitb777 committed:
Merge pull request #1047 from mattwojo/develop

Remove all lang-loc files

Commit:
Update IoTDashboard.md

IoTGirl committed:
Merge pull request #1048 from jasongottschalk/patch-2

Fixed typo - Confirmed

merzink committed:
Updated Link for DB Setup

Updated Link for DB Setup

rachitb777 committed:
Merge pull request #1050 from merzink/patch-1

Updated Link for DB Setup

aktsuda committed:
Add Windows Firewall settings to IoTDashboard.md

Add Windows Firewall settings to IoTDashboard.md because without this settings we cannot see any IoT device which auto-connecting through Wi-Fi. This change is allowed by Rachit Bajpai (rachitb), but he and/or his team will review this.

rachitb777 committed:
Some fixes in dashboard firewall settings page

rachitb777 committed:
Merge branch 'aktsuda-patch-1' into develop

rachitb777 committed:
Merge pull request #1053 from rachitb777/develop

Add Windows Firewall settings to IoTDashboard.md because without this settings we cannot see any IoT device which auto-connecting through Wi-Fi.

ooeygui committed:
Add NFC for IoT Sample and walkthrough

ooeygui committed:
Formatting

Commit:
15026 SDS release notes

rachitb777 committed:
Merge pull request #1057 from zeeshanfurqan/develop

15026 SDS release notes

Commit:
Fixing Dragonboard ISO name

Fixing Dragonboard ISO name

rachitb777 committed:
Merge pull request #1058 from parameshbabu/develop

Fixing Dragonboard ISO name

Commit:
Update NetworkPacketCapture.md

#1044

mattwojo committed:
Remove #iot_iot_home custom class 

The class was overriding expected UHF header behavior on hover noticeable now with the new UHF universal grey color.

mattwojo committed:
Added custom UHF hover color to match new grey styling

IoTGirl committed:
Merge pull request #1060 from cyberh0me/patch-1

Update NetworkPacketCapture.md - Merging based on Jordan's review. Thanks for the pull, Andre!

tiagoshibata committed:
Update Lightning performance tests numbers

ooeygui committed:
Merge pull request #1056 from ooeygui/develop

Add NFC for IoT Sample and walkthrough

tiagoshibata committed:
Add IoTAudioControlTool docs, fix typo in command

Commit:
Add note about some browsers duplicating requests

msftclas commented:
Hi __@nbaulesglobalsolutions__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

IoTGirl committed:
Merge pull request #1063 from munyirik/develop

Add note about some browsers duplicating requests

Commit:
Update FTP.md

take ownership and change file permissions to avoid access is denied during editing

IoTGirl committed:
Merge pull request #1067 from cyberh0me/patch-2

Update FTP.md

MahmoudGSaleh committed:
Merge pull request #1066 from tiagoshibata/develop

Update Lightning performance numbers and add IoTAudioControlTool docs

ooeygui commented:
Hi there,
We can't accept this change as is. It looks like it has many commit sandwiches. Please rebase and submit the specific change you are interested in pulling.
